libs/cart/src/selectors/cart/cart.selector.ts

```diff
@@ -163,6 +165,25 @@ const createCartSelectors = <
 		}
   );
 
+  const selectCanPlaceOrder = createSelector(
+    selectIsCartEmpty,
+    selectCartBillingAddress,
+    selectCartShippingAddress,
+    selectCartShippingInformation,
+    selectCartPayment,
+    (
+      isCartEmpty,
+      billingAddress,
+      shippingAddress,
+      shippingMethod,
+      paymentMethod
+    ) => !isCartEmpty
+      && billingAddress
```

Comment:
These child functions should be deferred into their associated selectors features.
e.g. 
```
hasBillingAddress
hasPaymentMethod
hasShippingAddress
hasShippingMethod
```

Reply:
changed